cam0:
  T_cam_imu:
  - [0.012082801900495658, -0.9997535103501118, -0.01862590778623028, 0.06343352451660102]
  - [-0.9999204045516665, -0.012148282517332184, 0.003406434075303824, -0.006856236700336373]
  - [-0.0036318672144901405, 0.018583265980630433, -0.9998207198123213, -0.015368579421887374]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [-0.43739662815925584, 0.24946478488129467, 0.00026227497009753773,
    0.002082318696185026]
  distortion_model: radtan
  intrinsics: [1021.6450540349482, 1021.7890266971525, 371.1525478172645, 288.06683762702767]
  resolution: [752, 480]
  rostopic: /cam0/image_raw
cam1:
  T_cam_imu:
  - [0.009925575897512537, -0.9999059368745232, -0.009465745939180259, -0.036990181623438076]
  - [-0.9999501382070789, -0.009935512881010791, 0.0010033361527445953, -0.007263358877851315]
  - [-0.0010972888165172645, 0.009455315270981611, -0.9999546954589393, -0.012443012967601627]
  - [0.0, 0.0, 0.0, 1.0]
  T_cn_cnm1:
  - [0.999955707288533, 0.0021901095055452346, -0.009153517435766092, -0.10054655714777176]
  - [-0.002167823597069914, 0.9999946639434122, 0.002443895570106552, -0.00023208686832158513]
  - [0.00915882099099771, -0.0024239441122517608, 0.9999551192393581, 0.002327281270293683]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [-0.43280121896614066, 0.24335641182486242, 0.000605408075761733,
    0.0023356072426615037]
  distortion_model: radtan
  intrinsics: [1019.4699748667822, 1020.338342231977, 346.1838623225366, 240.89924166392532]
  resolution: [752, 480]
  rostopic: /cam1/image_raw
T_imu_body:
  [1.0000, 0.0000, 0.0000, 0.0000,
  0.0000, 1.0000, 0.0000, 0.0000,
  0.0000, 0.0000, 1.0000, 0.0000,
  0.0000, 0.0000, 0.0000, 1.0000]
